server/services/xiaomi/lib/event/xiaomi.addMagnetSensor.js

```diff
@@ -0,0 +1,72 @@
+const logger = require('../../../../utils/logger');
+const { EVENTS } = require('../../../../utils/constants');
+/**
+ * @description Add node
+ * @param {number} sid - Id sensor.
+ * @param {boolean} closed - Closed magnet sensor.
+ * @param {number} battery - Battery sensor.
+ * @example
+ * addMagnetSensor(true);
+ */
+async function addMagnetSensor(sid, closed) {
+  logger.debug(`Xiaomi : set RAM variable and update value`);
+  this.magnetSensor[sid] = {
```

Comment:
Je réitère ce que j'avais dis dans ma précédente review, pourquoi ne pas stocker toutes les capteurs dans un même objet? J'ai l'impression qu'ici tu te complique la vie partout! Si on avait un objet type "this.devices" ou "this.sensors", ça serait bien plus simple partout. 

Quand on voudra faire du filtering poussé comme c'est le cas dans d'autres parties de Gladys, ça sera un enfer ici, alors que si c'est juste un seul objet c'est beaucoup plus simple.

Reply:
Bah pour moi c'est pas trop lourd d'avoir toujours les mêmes fonctions. L'objectif ici est tout simplement de laisser la possibilité de récupérer uniquement les capteurs de températures, uniquement tel ou tel device. 

Avec toujours une route général qui permet de tout récupérer. Mais si c'est pas bien je peux tout simplement faire comme tu dis tout mettre dans une seule variable. 

Je me suis dis autant laisser cette possibilité pour des besoins potentiels futurs.

Reply:
Je change du coup et mets tout dans un seul objet.